completions/meson.build

```diff
@@ -0,0 +1 @@
+install_data(sources : 'playerctl.bash', install_dir : '/etc/bash_completion.d')
```

Comment:
This should be optional and probably doesn't go here. Package maintainers might have some guidelines for how to include these.

I like [this example](https://github.com/swaywm/sway/blob/f79f00dbfb678c25e4c9ed423ef808fded312c33/meson.build#L241).